lib/active_record_doctor/detectors/missing_unique_indexes.rb

```diff
@@ -25,13 +25,16 @@ def message(model:, table:, columns:, problem:)
           "add a unique index on #{table}(#{columns.join(', ')}) - validating uniqueness in the model without an index can lead to duplicates"
         when :has_ones
           "add a unique index on #{table}(#{columns.first}) - using `has_one` in the #{model.name} model without an index can lead to duplicates"
+        when :habtms
```

Comment:
Let's use the fully spelled out name. I prefer to avoid abbreviations as they often introduce an extra step in understanding what is meant.